pixeltable/utils/formatter.py

```diff
@@ -0,0 +1,224 @@
+import base64
+import html
+import json
+import logging
+import mimetypes
+from typing import Any, Callable, Optional
+
+import PIL
+import PIL.Image as Image
+import cv2
+import numpy as np
+
+import io
+import pixeltable.type_system as ts
+from pixeltable.utils.http_server import get_file_uri
+
+_logger = logging.getLogger('pixeltable')
+
+_FLOAT_PRECISION = 3
+_LIST_THRESHOLD = 16
+_LIST_EDGEITEMS = 6
+_STRING_SEP = ' ...... '
+_STRING_MAX_LEN = 1000
+_NESTED_STRING_MAX_LEN = 300
+
+
+class PixeltableFormatter:
+    """
+    A factory for constructing Pandas HTML formatters for Pixeltable data. The formatters are used to customize
```

Comment:
If the scope is explicitly pandas, why not call this PandasFormatter (and get_formatter())?